README.md

```diff
@@ -1,5 +1,8 @@
 # Context
 
+![SROS2 CI](https://github.com/ros2/sros2/workflows/SROS2%20CI/badge.svg)
+[![codecov](https://codecov.io/gh/ros2/sros2/branch/master/graph/badge.svg)](https://codecov.io/gh/ros2/sros2)
```

Comment:
`SROS2 CI` badge does not currently render. I guess that will be fixed after merge and a first CI run?

Reply:
Indeed. Check [this](https://github.com/mikaelarguedas/sros2/workflows/SROS2%20CI/badge.svg) out to see the current state of it.